.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--
+
+Thanks for filing an issue! If this is a question or feature request, just delete
+everthing here and write out the request, providing as much context as you can.
+
+-->
+
+### What version of Go (`go version`) and `dep` are you using?
```

Comment:
Maybe it's time we gave #209 some love? 😁
`git describe --tags` has to be run within `dep` directory within GOPATH to report `dep` version and that isn't clear for many people and they might end up reporting their git repo tags (or worse report an error if their git repo doesn't have tags).